packages/protocol/test/backward/version.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  ContractVersion,
+  ContractVersionDelta,
+  Delta,
+  DeltaUtil,
+} from '@celo/protocol/lib/backward/version'
+import { assert } from 'chai'
+
+describe('#version()', () => {
+  describe('ContractVersion', () => {
+    it('isValid fails on invalid string', () => {
+      assert.isFalse(ContractVersion.isValid('1.2.3'))
+      assert.isFalse(ContractVersion.isValid('1.2'))
+      assert.isFalse(ContractVersion.isValid('1.2.3.5.'))
+      assert.isFalse(ContractVersion.isValid('test'))
+      assert.isFalse(ContractVersion.isValid('1.a.3.4'))
+    })
+    it('isValid true on valid string', () => {
+      assert.isTrue(ContractVersion.isValid('1.2.3.4'))
+      assert.isTrue(ContractVersion.isValid('-1.0.9.787'))
+      assert.isTrue(ContractVersion.isValid('10.0.224.9'))
+    })
+  })
+  describe('with a new ContractVersion instance', () => {
```

Comment:
Newline before `describe` (and similar throughout; may have also missed the same stuff in previous test file?)